File: src/main/java/com/muxiu1997/sharewhereiam/network/NetworkHandler.java
package com.muxiu1997.sharewhereiam.network;


import com.muxiu1997.sharewhereiam.Tags;
import cpw.mods.fml.common.network.NetworkRegistry;
import cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper;


public class NetworkHandler {
    public static final SimpleNetworkWrapper INSTANCE = NetworkRegistry.INSTANCE.newSimpleChannel(Tags.MODID);
}

File: src/main/kotlin/com/muxiu1997/sharewhereiam/network/NetworkHandler.kt
@file:JvmName("NetworkHandler")

package com.muxiu1997.sharewhereiam.network

import com.muxiu1997.sharewhereiam.Tags
import cpw.mods.fml.common.network.NetworkRegistry
import cpw.mods.fml.common.network.simpleimpl.SimpleNetworkWrapper

@JvmField
val network: SimpleNetworkWrapper = NetworkRegistry.INSTANCE.newSimpleChannel(Tags.MODID)
